### PR TITLE
Add Amplify Demo

### PR DESCRIPTION
**Pull request by michhyun1**
**Issue #:**

**Description of changes:**
Adds a demo to demonstrate how to create an Amazon Chime SDK full stack application using the Amazon Chime SDK React Component Library and AWS Amplify.

**Testing**

1. How did you test these changes? manually
2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it? n/a

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/AmplifyDemo/README.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    ? Select which capability you want to add: Lambda function (serverless function)
+    ? Provide an AWS Lambda function name: reactSampleLambda 
+    ? Choose the runtime that you want to use: NodeJS
+    ? Choose the function template that you want to use: Hello World
+    ? Do you want to configure advanced settings? No
+    ? Do you want to edit the local lambda function now? Yes
+    ```
+
+    When asked “Do you want to edit the local lambda function now?,” answer ‘Yes’ and replace the `./amplify/backend/function/reactSampleLambda/src/index.js` with the provided code in `./lambda/index.js`.
```
Comment (simmkyu) [inline apps/AmplifyDemo/README.md:67]: Fix markdown formatting. The above code block includes this question.

image

File: apps/AmplifyDemo/README.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    ? Enter a description for the API key:
+    ? After how many days from now the API key should expire (1-365): 7
+    ? Do you want to configure advanced settings for the GraphQL API No, I am done.
+    ? Do you have an annotated GraphQL schema? Yes
+    ```
+
+    When prompted for the schema.graphql “Do you have an annotated GraphQL schema?” - answer yes. Then, provide the file path to schema.graphql. (If you just cloned the repo and are running this command at the root level of this repo, you can just put `schema.graphql`).
+
+#####3. Create the cloud resources in your AWS account by running the following command. 
+This updates your cloud resources based on your current local environment’s Amplify configuration. Select all of the default options by pressing enter for each prompt. When asked if you’d like to continue, answer ‘yes'.
```
Comment (simmkyu) [inline apps/AmplifyDemo/README.md:87]: nit: `‘yes'` => `‘yes’` (trailing quote)

File: apps/AmplifyDemo/lambda/index.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
+/* Amplify Params - DO NOT EDIT
+	API_CHIME_GRAPHQLAPIENDPOINTOUTPUT
+	API_CHIME_GRAPHQLAPIIDOUTPUT
+	API_CHIME_GRAPHQLAPIKEYOUTPUT
+	ENV
+	REGION
+Amplify Params - DO NOT EDIT */
+
+var AWS = require('aws-sdk');
```
Comment (simmkyu) [inline apps/AmplifyDemo/lambda/index.js:9]: `var` => `const`

File: apps/AmplifyDemo/lambda/index.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  return response(200, 'application/json', JSON.stringify(
+    {
+      Attendee: attendeeInfo.Attendee
+    }, null, 2));
+}
+
+const endChimeMeeting = async (context) => {
+  const meetingId = context.arguments.meetingId;
+  await chime.deleteMeeting({ MeetingId: meetingId });
+  console.log("Deleted Meeting: " + meetingId);
```
Comment (simmkyu) [inline apps/AmplifyDemo/lambda/index.js:105]: nit: Use single quotes for consistency with other strings. Or you can run ESLint or Formatter once to fix all minor syntaxes like missing semicolon for an arrow function.

File: apps/AmplifyDemo/src/components/MeetingForm.tsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+import React, { ChangeEvent, FC, FormEvent, useState } from 'react';
+
+import {
+  Flex,
+  FormField,
+  Input,
+  PrimaryButton,
+  useMeetingManager,
+} from 'amazon-chime-sdk-component-library-react';
+import { addAttendeeToDB, addMeetingToDB, createMeeting, getAttendeeFromDB, getMeetingFromDB, joinMeeting } from '../utils/api';
```
Comment (simmkyu) [inline apps/AmplifyDemo/src/components/MeetingForm.tsx:13]: Recommend formatting them in multiple lines like the above one (`amazon-chime-sdk-component-library-react`).

File: apps/AmplifyDemo/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```
Comment (simmkyu) [inline apps/AmplifyDemo/tsconfig.json:3]: You may target a later version, e.g., [ES2015 in the JS demo](https://github.com/aws/amazon-chime-sdk-js/blob/2b133458cfae7ff45aeb942bb8e2fc19b23534cd/config/tsconfig.base.json#L17), or [even later one like React demo](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/dfe904fec85b5f7c21af690e37c32e4806320e81/demo/meeting/tsconfig.json#L10)

**Review by simmkyu — CHANGES_REQUESTED** [review]
The code looks good to me overall. For readme, you can see the preview to fix formatting.

File: apps/AmplifyDemo/README.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+    ? Enter a name for the environment: dev 
+    ? Choose your default editor: Visual Studio Code
+    ? Choose the type of app that youre building: javascript 
+    ? What JavaScript framework are you using: React
+    ? Source Directory Path: src 
+    ? Distribution Directory Path: dist 
+    ? Build Command: npm run-script build
+    ? Start Command: npm run-script start 
+    ? Do you want to use an AWS profile? Yes 
+    ? Please choose the profile you want to use [Your AWS Profile] <- Choose the AWS Profile configured in the previous step
```
[the remark this answers is not in the document]
Comment (michhyun1) [reply apps/AmplifyDemo/README.md:39]: This is the 'amplify configure steop in the pre-requisite step, but I'll move the configure step into this section.

**Comment by michhyun1** [comment]
> The code looks good to me overall. For readme, you can see the preview to fix formatting.
> 
> 

Centered the image now

**Comment by simmkyu** [comment]
6 hidden comments are missing :)

**Comment by michhyun1** [comment]
> 6 hidden comments are missing :)
> 
> 

Didnt see that, thanks. Fixed all of the outstanding comments

**Review by simmkyu — APPROVED** [review]

**Pull request merged**